api/models.py

```diff
@@ -16,18 +16,25 @@
 
 # Users Models Class:
 class UsersModel(db.Model):
+    """This is the model for users in the database.
+
+    For now, this only contains admin information.
+    """
     __tablename__ = 'users'
 
     # Table attributes:
     id = db.Column(db.Integer, primary_key=True)
     username = db.Column(db.String(MAX_CREDENTIAL_LEN))
     email = db.Column(db.String(MAX_CREDENTIAL_LEN))
     password = db.Column(db.String(MAX_CREDENTIAL_LEN))
+    is_admin = db.Column(db.Boolean)
 
-    def __init__(self, username: str, email: str, password: str):
+    def __init__(self, username: str, email: str, password: str,
```

Comment:
Does `is_admin` have a default set on the database (probably should be false)? If not, might want to set that here.